docusaurus/website/docusaurus.config.base.js

```diff
@@ -145,6 +145,17 @@ const themeConfigFooter = {
 const themeConfigPrism = {
   theme: grafanaPrismTheme,
   additionalLanguages: ['bash', 'diff', 'json'],
+  magicComments: [
+    {
+      className: 'theme-code-block-highlighted-line',
+      line: 'highlight-next-line',
+      block: { start: 'highlight-start', end: 'highlight-end' },
+    },
```

Comment:
Am I correct that this currently doesn't have the associated styling (css class) for it? If that's true, should we keep it? (Another idea is to have this block definition for the `addition-highlight-next-line` as well.)

Reply:
You are indeed correct! I'll figure out how we can cater for next line and block. 👍